webapp/channels/src/components/user_list_row_with_error/user_list_row_with_error.tsx

```diff
@@ -89,12 +88,13 @@ export default class UserListRowWithError extends React.PureComponent<Props, Sta
         if (this.props.user.is_bot) {
             email = null;
         } else if (this.props.extraInfo && this.props.extraInfo.length > 0) {
+            // Replaced FormattedMarkdownMessage with FormattedMessage
             email = (
-                <FormattedMarkdownMessage
-                    id='admin.user_item.emailTitle'
+                <FormattedMessage
+                    id='admin.user_item.emailTitle_updated'
                     defaultMessage='**Email:** {email}'
                     values={{
-                        email: this.props.user.email,
+                        email: <strong>{this.props.user.email}</strong>, // Ensure rich text formatting
```

Comment:
This changes the description!
We need the text email in strong, while the email as be as is

Reply:
i guess i missed that slightly

Reply:
Can you take a look at the example PR https://github.com/mattermost/mattermost/pull/28438

Reply:
Check if it works now or let me know exact details

Reply:
it is expected that you run the app and check out its where this component is used and verify it its same as before. 

We would like to keep the same text structure as before but by using the different component.

Reply:
i kept the same structure in the latest commit please check